tests/gtfs2ntfs.rs

```diff
@@ -204,7 +204,31 @@ fn test_minimal_gtfs_with_odt_comment() {
         compare_output_dir_with_expected(
             &path,
             Some(vec!["comment_links.txt", "comments.txt", "stop_times.txt"]),
-            "./tests/fixtures/gtfs2ntfs/odt_comment",
+            "./tests/fixtures/gtfs2ntfs/odt_comment/without_frequencies",
```

Comment:
Could you prefix the 2 folders with "output_" ?
It's easier when we navigate in fixtures.